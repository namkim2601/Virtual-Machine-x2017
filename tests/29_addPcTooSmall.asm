FUNC LABEL 0
    MOV REG 0 VAL 3
    MOV REG 1 VAL 5
    ADD REG 0 REG 1
    NOT REG 0
    ADD REG 7 REG 0
    RET
