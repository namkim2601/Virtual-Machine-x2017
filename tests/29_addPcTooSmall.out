Error: Program counter value is out of scope of instructions Func 0, PC 4
